frontend/packages/jobs/package.json

```diff
@@ -2,7 +2,9 @@
   "name": "@liam-hq/jobs",
   "private": true,
   "version": "0.1.0",
-  "type": "module",
+  "publishConfig": {
+    "access": "restricted"
+  },
```

Comment:
The following changes are being reverted.
https://github.com/liam-hq/liam/commit/2311c9aaa161a2ecde0848f4f0810bac91aedda9

![ss 3253](https://github.com/user-attachments/assets/31518750-1abf-40cc-b486-9ed9cb722701)